cmd/contour/servecontext_test.go

```diff
@@ -338,7 +339,8 @@ func TestServeContextCertificateHandling(t *testing.T) {
 	checkFatalErr(t, err)
 
 	// Start a dummy server.
-	opts := ctx.grpcOptions()
+	log := logrus.StandardLogger()
+	opts := ctx.grpcOptions(log)
```

Comment:
Since this is a test, you can use the test logger to capture any logs:
```Go
opts := ctx.grpcOptions(fixture.NewTestLogger(t))
```

Reply:
Done.